src/pages/MapViewer/components/Toolbox/components/item-assets/AssetsPanel.scss

```diff
@@ -0,0 +1,12 @@
+.asset-row {
+  width: 100%;
+  display: grid;
+  grid-template-columns: 140px 1fr auto;
+  grid-gap: 0.5rem;
+  padding: 0.5rem;
+
+  & .label {
```

Comment:
Could omit the `&`

Reply:
your right, I will remove this. Been a few years since I used SASS, but rusty....